recipes/dcmstack/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "v0.6.2" %}
+
+package:
+  name: dcmstack
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/moloney/dcmstack.git
+  git_tag: {{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - dcmstack
+  commands:
+    - python -c "import example;"
```

Comment:
Hoping this is not just `example`. Why can't this just be part of the regular import tests?